modules/web3/constants/networks.ts

```diff
@@ -79,20 +81,6 @@ export const CHAIN_INFO: ChainInfo = {
     },
     showInProduction: false
   },
-  [SupportedChainId.GOERLIFORK]: {
-    blockExplorerUrl: 'goerli.etherscan.io',
-    blockExplorerName: 'Etherscan',
-    chainId: SupportedChainId.GOERLIFORK,
-    label: 'GoerliFork',
-    type: 'normal',
-    network: SupportedNetworks.GOERLIFORK,
-    defaultRpc: NodeProviders.LOCAL,
-    spockUrl: LOCAL_SPOCK_URL,
-    rpcs: {
-      [NodeProviders.LOCAL]: 'http://127.0.0.1:8545/'
-    },
-    showInProduction: false
-  },
   [SupportedChainId.ARBITRUMTESTNET]: {
```

Comment:
Do we need to update this from goerli to sepolia?

Reply:
yep will do